rest/route/project.go

```diff
@@ -393,6 +404,14 @@ func (h *projectIDPatchHandler) Run(ctx context.Context) gimlet.Responder {
 		return gimlet.MakeJSONErrorResponder(errors.Wrapf(err, "Database error deleting subscriptions for project '%s'", h.projectID))
 	}
 
+	after, err := h.sc.GetProjectSettingsEvent(dbProjectRef)
+	if err != nil {
+		return gimlet.MakeJSONErrorResponder(errors.Wrapf(err, "Error getting ProjectSettingsEvent after update for project'%s'", h.projectID))
+	}
+	if err = dbModel.LogProjectModified(identifier, h.username, *before, *after); err != nil {
```

Comment:
You may want to verify that before and after aren't nil, though this shouldn't happen.

Reply:
Or if it's only being used here, better to change the project signature to take pointers.

Reply:
Ping on this ^^ I think LogProjectModified can just take pointers.

Reply:
`LogProjectModified` now takes pointers